site/docs/projection.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+menu: docs
+title: Projection
+permalink: /docs/projection.html
+---
+## Projection Overview
+Projection's are specified at the mark specification level, alongside encoding.
+
+{:#properties}
+## Projection Properties
+{% include table.html props="type,clipAngle,clipExtent,center,rotate,precision" source="Projection" %}
```

Comment:
From Vega docs, 

> In addition to the shared properties above, the following properties are supported for specific projection types in the d3-geo-projection library: coefficient, distance, fraction, lobes, parallel, radius, ratio, spacing, tilt.

Do we support these properties too?

Reply:
We don't currently. Do you want to? I can add them in `projection.ts`

Reply:
I think so.